ups/meas_base.cfg

```diff
@@ -21,5 +21,5 @@ config = lsst.sconsUtils.Configuration(
     __file__,
     headers=["lsst/meas/base.h"],
     hasDoxygenInclude=False,
-    hasSwigFiles=True,
+    hasSwigFiles=False,
```

Comment:
Noted, but in that case we should go back and delete this line from all the lower-level packages. :wink: